From 5e3f8a20f628198bfb5cc6fb68e426de3c57f11b Mon Sep 17 00:00:00 2001
From: Kaleb S KEITHLEY <kkeithle@redhat.com>
Date: Fri, 6 May 2016 12:16:10 -0400
Subject: [PATCH 139/139] packaging: additional dirs and files in /var/lib/glusterd/

Directory /var/lib/glusterd/glusterfind/.keys doesn't need
to be created, it just needs to be listed in the list of files.

Also, S57glusterfind-delete-post.py under the hooks directory
is actually a symlink to libexec/glusterfs/glusterfind/...
so don't list %attr(...) for the symlink

Change-Id: Ic8e20215f30a9bf0b34f3f790062dbda98ca1917
BUG: 1324820
Signed-off-by: Kaleb S KEITHLEY <kkeithle@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/73831
Reviewed-by: Aravinda Vishwanathapura Krishna Murthy <avishwan@redhat.com>
---
 glusterfs.spec.in |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index f5bd905..15e25a9 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -799,8 +799,6 @@ find ./tests ./run-tests.sh -type f | cpio -pd %{buildroot}%{_prefix}/share/glus
 install -p -m 0744 -D extras/command-completion/gluster.bash \
     %{buildroot}%{_sysconfdir}/bash_completion.d/gluster
 
-mkdir -p %{buildroot}%{_sharedstatedir}/glusterd/glusterfind/.keys
-
 %clean
 rm -rf %{buildroot}
 
@@ -1323,7 +1321,7 @@ exit 0
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/copy-file/pre
        %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/delete
        %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/delete/post
-            %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/delete/post/S57glusterfind-delete-post.py
+                            %{_sharedstatedir}/glusterd/hooks/1/delete/post/S57glusterfind-delete-post.py
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/delete/pre
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/remove-brick
 %ghost %dir %attr(0755,-,-) %{_sharedstatedir}/glusterd/hooks/1/remove-brick/post
-- 
1.7.1

